.github/workflows/integration.yml

```diff
@@ -72,7 +72,7 @@ jobs:
                         "30",
                         "35",
                         "40",
-                        # "50", # times out
+                        "50",
```

Comment:
But I'm not sure if we should re-enable test 50, although it's working now with the NetworkManager, it still fails with the network-legacy module:

https://github.com/dracutdevs/dracut/actions/runs/4076254978/jobs/7075949251

Reply:
Okay